Staging Merge: Latest Code from Development to Staging

supriyartambewaiin opened the pull request:
No Screenshots: 12/02/2025

alok-nakhate reviewed (approved):
Changes are reviewed and approved.

nitinsonawane123 committed:
Commit Changes for bug 1327

SoheshMG committed:
reopen rds-label p-0 m-0

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1327-component-dialog-cross-button-alignment-issue

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1327-component-dialog-cross-button-alignment-issue

rameshwarkamandar committed:
Resolved bug1440: Enhancement & remove unnecessary console.log.

rameshwarkamandar committed:
Merge branch 'development' into users/rameshwar/bug/1440-elements-ai-prompt-box-Enhancement-&-remove-unnecessary-console.log

rameshwarkamandar committed:
Merge pull request #1441 from Wai-Technologies/users/rameshwar/bug/1440-elements-ai-prompt-box-Enhancement-&-remove-unnecessary-console.log

Bug 1440: Design System > Element > AI prompt Box > Default > Enhancement & remove unnecessary console.log.

Dipak7028 committed:
Resolved Bug 1370 : Design System > Components > Login > Default > Fixed Language Dropdown

Resolved Element level problem of Dropdownlist in dropdown.scss
Able to resolve placement issue of Select Language

Dipak7028 committed:
Resolved Bug 1147: Design System Components Url Forwardings Default Required message is not display

As Per Required message display in Resolved

Dipak7028 committed:
Merge pull request #1442 from Wai-Technologies/users/dipak/bug-1370-design-system-Components-login-default-language-dropdown-issue-position-is-not-fixed

Resolved Bug 1370 : Design System > Components > Login > Default > Fixed Language Dropdown

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1147-design-system-urlforwardings-required-required-message-is-not-display

Dipak7028 committed:
Resolved Bug 1147: Design System Components Url Forwardings Default Required message is not display

Resolved issue Required message is not display
Resolved issue Language dropdown issue

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1389-design-system-elements-review-category-center-aligned-left-aligned-stories-profile-icon

SoheshMG committed:
Changed rating stars gap

nitinsonawane123 committed:
Commit Changes for bug 1261

nitinsonawane123 committed:
Merge pull request #1445 from Wai-Technologies/users/nitin/bug-1261-element-breadcrumb-state-is-missing-as-per-the-wireframe

Resolved bug 1261 : Design System > Elements > Breadcrumb: - State is missing as per wireframe & compare Breadcrumb controls & UI as per Figma.

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1147-design-system-urlforwardings-required-required-message-is-not-display

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1327-component-dialog-cross-button-alignment-issue

supriyartambewaiin committed:
Merge branch 'development' into users/sohesh/bug-1389-design-system-elements-review-category-center-aligned-left-aligned-stories-profile-icon

nitinsonawane123 committed:
Commit Changes for bug 1327

Dipak7028 committed:
Committed changes on removing commented code

Committed changes on removing commented code

Dipak7028 committed:
Merge pull request #1446 from Wai-Technologies/users/dipak/bug-1147-design-system-urlforwardings-required-required-message-is-not-display

Users/dipak/bug 1147 design system urlforwardings required required message is not display

nitinsonawane123 committed:
Commit Changes for bug 1327

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1327-component-dialog-cross-button-alignment-issue

VenuKG committed:
commit changes for add download option to file uploader

nitinsonawane123 committed:
Merge pull request #1407 from Wai-Technologies/users/nitin/bug-1327-component-dialog-cross-button-alignment-issue

Resolved bug 1327 : Design System > Components >Dialog > Default > Cross button alignment issue.

supriyartambewaiin committed:
Merge branch 'development' into users/venu/design-system-file-uploader-add-download-option

supriyartambewaiin committed:
Merge pull request #1449 from Wai-Technologies/users/venu/design-system-file-uploader-add-download-option

Completed Task : Design System-> File Uploader -> add download option to file uploader

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1389-design-system-elements-review-category-center-aligned-left-aligned-stories-profile-icon

SoheshMG committed:
Merge pull request #1444 from Wai-Technologies/users/sohesh/bug-1389-design-system-elements-review-category-center-aligned-left-aligned-stories-profile-icon

Resolved Bug 1389 : Design System > Elements > Review Category > Center Aligned/Left Aligned Stories Profile Icon Is Enlarged Reopen issue solved

Dipak7028 committed:
Resolved Bug 823 : Design System > Create color variants for toast elements

Dipak7028 committed:
Update rds-toast.tsx

Dipak7028 committed:
Merge pull request #1453 from Wai-Technologies/users/dipak/bug-823-design-system-create+color-variants-for-toast-elements

Resolved Bug 823: Design System > Create color variants for toast elements.

supriyartambewaiin committed:
Merge branch 'staging' into users/supriya/merge-development-to-staging-for-deployment
